frontend/packages/client/src/components/Community/CommunityPropsAndVoting.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ActionButton from 'components/ActionButton';
+import StrategySelectorForm from 'components/Community/StrategySelectorForm';
+
+// this object is used to change field names as are used
+// on the backend
+const fieldMapPayload = {
+  contractAddress: 'addr',
+  contractName: 'name',
+  maxWeight: 'maxWeight',
+  minimunBalance: 'threshold',
+  publicPath: 'publicPath',
+};
+export default function CommunityProposalsAndVoting({
+  communityVotingStrategies = [],
+  updateCommunity,
+  updatingCommunity,
+} = {}) {
+  // sends updates to backend
+  const saveData = async (strategies) => {
+    const updatePayload = strategies.map((st) => ({
+      name: st.strategy,
+      // only other strategies than 'one-address-one-vote' have contract information
+      ...(st.strategy !== 'one-address-one-vote'
+        ? {
+            contract: Object.assign(
+              {},
+              ...Object.entries(st).map(([key, value]) => ({
+                ...(fieldMapPayload[key] && value
+                  ? {
+                      [fieldMapPayload[key]]: value,
+                    }
+                  : undefined),
+              }))
+            ),
+          }
+        : undefined),
+    }));
+    await updateCommunity({
+      strategies: updatePayload,
+      name: 'this field should not be here',
```

Comment:
@rvmelkonian this is the field we need to send to make the request to work, as we spoke should be removed as mandatory.

Reply:
the duplicate field?

Reply:
yeah, at some point sending the name was required, otherwise the request was failing, but now this is just: 
```
    await updateCommunity({
      strategies: updatePayload,
    });